UnityProject/Assets/Scripts/Systems/Inventory/Structure/DefinedSlotCapacity.cs

```diff
@@ -42,6 +42,8 @@ public class DefinedSlotCapacity : SlotCapacity
 	public override bool CanFit(Pickupable toCheck)
 	{
 		if (toCheck == null) return false;
+		bool ToReturn = true;
```

Comment:
```suggestion
		bool canFit = true;
```

Reply:
`ToReturn` can be renamed to `HasRequiredTraits`